### PR TITLE
add 2 versions of equality of predicates 

### LINKED ISSUE #1547: Predicate equivalence?
Is there any reason (other than no one has done it yet) that
```agda
 _≐_ : Pred 𝒜 ρ → Pred 𝒜 ρ → Type (α ⊔ ρ)
 P ≐ Q = (P ⊆ Q) × (Q ⊆ P)
```
isn't in `Relation.Unary` ? The fact that it's an equivalence relation should probably go in a to-be-created `Relation.Unary.Bundles` ?

### PR DESCRIPTION
**Pull request by JacquesCarette**
and the fact that they are equivalence relations.

Closes #1547.

### CODE REVIEW

File: src/Relation/Unary/Relation/Binary/Equality.agda [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+open import Level using (Level)
+open import Relation.Binary.Structures using (IsEquivalence)
+open import Relation.Unary using (_≐_; _≐′_)
+
+private
+  variable
+    a ℓ₁ ℓ₂ : Level
+    A : Set a
+
+≐-isEquivalence : IsEquivalence (_≐_ {A = A} {ℓ₁})
```
Comment (MatthewDaggitt) [inline src/Relation/Unary/Relation/Binary/Equality.agda:22]: I think these would be more readable as `IsEquivalence {A = Pred A ℓ₁} _≐_`?

File: src/Relation/Unary.agda [hunk 2]
```diff
@@ -70,7 +70,7 @@ x ∉ P = ¬ x ∈ P
 ------------------------------------------------------------------------
 -- Subset relations
 
-infix 4 _⊆_ _⊇_ _⊈_ _⊉_ _⊂_ _⊃_ _⊄_ _⊅_
+infix 4 _⊆_ _⊇_ _⊈_ _⊉_ _⊂_ _⊃_ _⊄_ _⊅_ _≐_
```
Comment (MatthewDaggitt) [inline src/Relation/Unary.agda:73]: Missing `_≐′_`?

File: src/Relation/Unary/Relation/Binary/Equality.agda (hunk 1 above)
Comment (JacquesCarette) [reply src/Relation/Unary/Relation/Binary/Equality.agda]: Fixed.

File: src/Relation/Unary.agda (hunk 2 above)
Comment (JacquesCarette) [reply src/Relation/Unary.agda]: Fixed.

**Pull request merged**